src/planning/angular/US1/pt/US1.md

```diff
@@ -0,0 +1,15 @@
+## Configuração do projeto
+
+## Descrição:
+
+Ao iniciar um projeto, sempre há uma série de tarefas nas quais precisamos dedicar tempo.
+
+## Definição de concluído
+
+- [ ] Devemos ser capazes de executar o comando *ng serve* e fazer com que nosso navegador mostre o aplicativo padrão de um projeto Angular.
+
+- [ ] Devemos ser capazes de executar o comando *ng test* para executar os testes (jest).
+
+- [ ] Não deve ser possível fazer um push direto para o branch principal.
+
+- [ ] Deve haver um Figma com layouts de alta fidelidade
```

Comment:
Em relação a este item, não acha que o layout de média fidelidade já seria o suficiente? 
Digo isso para que as estudantes possam focar mais do tempo delas na implementação do projeto.

Reply:
Task 1 limits to only 1 or 2 days to create a high fidelity prototype